javascript/ql/src/Security/CWE-078/IndirectCommandInjection.qhelp

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+	<overview>
+
+		<p>
+
+			Forwarding command-line arguments to
+			<code>require('child_process').exec</code>, or some other library
```

Comment:
```suggestion
			<code>child_process.exec</code>, or some other library
```

(This is how the Node.js API documentation refers to it. Also, not everyone uses `require` anymore.)